lib/vquic/msh3.c

```diff
@@ -131,13 +143,18 @@ static bool msh3request_ensure_room(struct HTTP *stream, size_t len)
 {
   uint8_t *new_recv_buf;
   const size_t cur_recv_len = stream->recv_header_len + stream->recv_data_len;
+
   if(cur_recv_len + len > stream->recv_buf_alloc) {
     size_t new_recv_buf_alloc_len = stream->recv_buf_alloc;
     do {
       new_recv_buf_alloc_len <<= 1; /* TODO - handle overflow */
     } while(cur_recv_len + len > new_recv_buf_alloc_len);
+    CF_DEBUGF(fprintf(stderr, "* enlarging buffer to %zu\n",
```

Comment:
Why are you sending these all to `stderr`?

Reply:
stderr is easier for debugging than stdout and Curl's own `infof()` cannot be called, since this is on the "wrong" thread.

Reply:
Ok.